todo/src/components/Todo.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { connect } from "react-redux";
+import { todos, addTodo, completedTodo } from "../actions/index";
+import "./Todo.css";
+class Todo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newTodo: ""
+    };
+  }
+
+  handleNewTodoInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+  handleAddNewFriend = e => {
+    e.preventDefault();
+    this.props.addTodo(this.state.newTodo);
+    this.setState({ newTodo: "" });
+  };
+  render() {
+    console.log(this.props);
+    return (
+      <div className="todo-container">
+        {this.props.todos.map((todo, index) => (
+          <div>
```

Comment:
```suggestion
          <div key={index}>
```

Reply:
Member that the rule of thumb is that the first element after we used the map method, should have the key on it